tox.ini

```diff
@@ -33,6 +33,7 @@ deps =
 deps = -r{toxinidir}/docs/requirements.txt
 changedir = docs
 commands = sphinx-build -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
+           sphinx-build -b linkcheck -d {envtmpdir}/doctrees . {envtmpdir}/html
```

Comment:
Can we add a new testenv, docs-linkcheck, to run this command? I guess it takes quite some time to run, so I'm not sure I want Travis builds to be delayed because of the linkcheck. Having it as an independent testenv in tox that `.travis.yml` don't know about is still helpful, because it will be run when a developer runs `tox` without arguments in his dev env.